RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs

```diff
@@ -1132,25 +1126,14 @@ protected void btnSave_Click( object sender, EventArgs e )
                                 return;
                             }
 
-                            if ( _isFamilyGroupType )
-                            {
-                                if ( person.RecordStatusValueId != recordStatusValueID )
```

Comment:
I'm not sure I understand why this code was removed.  This case occurs when adding an existing person into the edited family and when their _current/old_ record status is different than what is being set on the edit form.